eventarc_workflows/workflow.yaml

```diff
@@ -0,0 +1,17 @@
+# This is a sample workflow that simply logs the incoming Pub/Sub event
+# Note that $$ is needed for Terraform
```

Comment:
Per [Hashicorp documentation](https://www.terraform.io/language/functions/templatefile) the preferred file format for templates is `.tftpl`  

That will help avoid any confusion re: the double $'s

Reply:
This was copied as is from MM, I opened an issue for a follow up PR.